src/processes.py

```diff
@@ -32,7 +32,7 @@ def main():
                 print(template.format(command, 'cpu_usage', process.pcpu))
                 print(template.format(command, 'mem_usage', process.pmem))
                 print(template.format(command, 'etimes', process.etimes))
-                print(template.format(command, 'rss', process.rss))
+                print(template.format(command, 'rss', int(process.rss) * 1024))
```

Comment:
@TuxPowered42 actually there is no moment of parsing.
https://github.com/innogames/igcollect/blob/ab9d42178de187e1189d158d27d822edc843f2fd/src/processes.py#L46-L47

Only marshalling.